behat.yaml.dist

```diff
@@ -0,0 +1,15 @@
+default:
```

Comment:
I think you mean example `behat.yml` file, not configuration for this feature (because we don't use Behat to test this project).

Because of this I think it will be better to rename file to `behat.yml.example`.